lib/commands/update.js

```diff
@@ -55,19 +55,28 @@ module.exports = {
 					if ( stdout ) {
 						throw new Error( `Package "${ data.packageName }" has uncommitted changes. Aborted.` );
 					}
-				} )
-				.then( () => {
+
```

Comment:
I don't like merging these callbacks. The log handling is a separate task which, btw, should be done differently (related to https://github.com/cksource/mgit2/issues/13, but not exactly this).